### PR TITLE
#5470 [Manager] Fix Toolbar not correctly reset opened panels state i…

### PR DESCRIPTION
**Pull request by konekoya**
…ssue

### Checklist

- [x] [Review Contribution Guidelines](https://oharastream.github.io/en/docs/master/contributing/)
- [x] Add at least one reviewer (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add an assignee (of course, you are the assignee by default)
- [x] Do not set milestone, projects and labels (PR labels are added automatically by Jenkins)
- [x] Link to an issue
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (If the failed tests are unrelated to this PR, add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/cypress/it/paper/toolbar.test.ts [hunk 1]
```diff
@@ -121,6 +121,27 @@ describe('Toolbar', () => {
       cy.get('#stream-list').should('be.visible');
       cy.get('.toolbox-list:visible').should('have.length', 1);
     });
+
+    it.only('should reset and open only one panels when more than one panel are opened', () => {
```
Comment (konekoya) [inline ohara-manager/client/cypress/it/paper/toolbar.test.ts:125]: oops

File: ohara-manager/client/src/components/Pipeline/Toolbar/Toolbar.js [hunk 2]
```diff
@@ -352,7 +351,7 @@ const Toolbar = (props) => {
 
 Toolbar.propTypes = {
   handleToolboxOpen: PropTypes.func.isRequired,
-  handleToolbarClick: PropTypes.func.isRequired,
+  handleToolbarInsertClick: PropTypes.func.isRequired,
```
Comment (eechih) [inline ohara-manager/client/src/components/Pipeline/Toolbar/Toolbar.js:354]:
```suggestion
  onInsertClick: PropTypes.func.isRequired,
```

Comment (konekoya) [reply ohara-manager/client/src/components/Pipeline/Toolbar/Toolbar.js:354]: @eechih Is this a naming convention we would like to apply to all our components?

Comment (eechih) [reply ohara-manager/client/src/components/Pipeline/Toolbar/Toolbar.js:354]: > Is this a naming convention we would like to apply to all our components?
I think this is better :)

Comment (konekoya) [reply ohara-manager/client/src/components/Pipeline/Toolbar/Toolbar.js:354]: This rule seems like a perfect fit for this :)
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-handler-names.md

Comment (eechih) [reply ohara-manager/client/src/components/Pipeline/Toolbar/Toolbar.js:354]: Yap

Comment (konekoya) [reply ohara-manager/client/src/components/Pipeline/Toolbar/Toolbar.js:354]: I will open another issue and so we can discuss there

Comment (eechih) [reply ohara-manager/client/src/components/Pipeline/Toolbar/Toolbar.js:354]: > This rule seems like a perfect fit for this :)
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-handler-names.md

👍👍

Comment (konekoya) [reply ohara-manager/client/src/components/Pipeline/Toolbar/Toolbar.js:354]: Let's continue our discssion in #5472 :)

**Comment by konekoya** [comment]
retry manager-e2e

**Pull request merged**